grace/base.py

```diff
@@ -102,16 +149,16 @@ def delaunay_edges_from_nodes(
     # add edge nodes
     if update_graph:
         edge_attrs = {
-            GraphAttrs.EDGE_PREDICTION: 0.0,
+            # GraphAttrs.EDGE_PREDICTION: (0, np.array([0.9, 0.1])),
```

Comment:
Should this be uncommented too?

Reply:
It doesn't have to be there as the `update_graph` only needs to construct the edges with UNKNOWN GT labels. Predictions can / will be added afterwards, if needed, either with the update dummy function or from real classifier.

Reply:
I took it out completely.